projects/plugins/jetpack/modules/blocks.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * Module Name: Blocks
+ * Module Description: Add additional blocks to your site and post editors.
+ * Sort Order: 5
+ * First Introduced: 13.9
```

Comment:
```suggestion
 * First Introduced: 13.9-a.8
```
Since the `$$next-version$$` thing doesn't (currently) work here.